Insights/Host/Top_Operations_and_Categories.yaml

```diff
@@ -0,0 +1,40 @@
+SchemaVersion: '1.0'
+DataTypes:
+- DataType: AzureActivity
+Provider: Sentinel
+Type: KQL
+ParentEntityQuery: https://github.com/Azure/Azure-Sentinel/blob/vm_insights/Insights/Azure-Resource/Top_Operations_and_Categories.yaml
+EntitiesFilter: {}
```

Comment:
You may use the OS filter, to prevent this insight appearing on non windows hosts' entity pages

Reply:
The query is running on the AzureActivity table, so it only takes Azure VMs, isn't it?

Reply:
What should I care if it's Windows or not? If I have AzureActivity records for this VM it's interesting, isn't it?